domains/heromax.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "tileicklest",
+        "email": "deshaunmcceay374@gmail.com"
+    },
+    "record": {
+        "CNAME": "heromax.is-a.dev"
```

Comment:
No site content, nothing is displayed. Redirects back to our homepage. This is the same domain which you are requesting which doesn't make sense.